fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -150,6 +151,45 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y, event=_ALL, **kwargs)
 
 
+class TruePositiveRateDifference(ConditionalSelectionRate):
+    r"""Implementation of Equal Opportunity as a moment.
+
+    Adds conditioning on label `y=1` compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = 1 | A = a, Y = 1] = P[h(X) = 1 | Y = 1] \; \forall a
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    the event corresponding to `y=1`.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to each unique value of `y = 1` in
+    the `Y` array.
+
+    The `index` MultiIndex will have a number of entries equal to
+    the number of unique values for the sensitive feature, multiplied by
+    the number of unique values of the `Y` array, which is just `label=1`
+    (1 unique value), multiplied by two (for the Lagrange multipliers
+    for positive and negative constraints).
+
+    With these definitions, the :meth:`signed_weights` method
```

Comment:
Funny typo there: meth -> math

Reply:
I think this is existing already and this means `method`. I cross-checked this the documentation of `EqualizedOdds` and it seems to be the same there too.

Reply:
:O OK the joke's on me! Also, what kind of funny abbreviation for method...